mne_bids/pick.py

```diff
@@ -0,0 +1,70 @@
+# Authors: Matt Sanderson <matt.sanderson@mq.edu.au>
+#
+# License: BSD (3-clause)
+
+from mne.io.constants import FIFF
+
+
+def get_coil_types():
+    """Return all known coul types.
```

Comment:
Why are you making a function to return a dictionary? Why not simply define the dictionary within the scope of the `pick` module?

Reply:
I was trying to keep with the same code style as was used in the `pick.py` file in mne-python.

Reply:
Mmh [I see](https://github.com/mne-tools/mne-python/blob/609515033e9917423aa4005c294a7c64919b21e6/mne/io/pick.py) ... that begs the question why it's done like that in MNE-Python though 🤔

Reply:
maybe @agramfort can enlighten us ...

Reply:
the function makes it so that the dict is recreated every function call which is not ideal but maybe not a big deal. Using a module variance such as `COIL_TYPES = dict(...)` would be the alternative.

no strong feeling on my side

Reply:
@monkeyman192 I vote for a `COIL_TYPES = dict()` solution instead of a function.